query.go

```diff
@@ -786,7 +789,17 @@ func scanRow(cols []string, rows *sql.Rows) (Result, error) {
 			case int:
 				result[cols[i]] = int64(v.(int))
 			case []byte: // text cols are given as bytes
-				result[cols[i]] = string(v.([]byte))
+				if columnTypes[i].DatabaseTypeName() == "JSONB" || columnTypes[i].DatabaseTypeName() == "JSON" {
+					var inter interface{}
+					err := json.Unmarshal(v.([]byte), &inter)
+					if err != nil {
+						result[cols[i]] = err.Error()
```

Comment:
If we have an error we should return it directly like line 773. We should not put the error in the result at all. Maybe we should return the string representation of the byte as in line 801 but for sure not the error.

Reply:
FIXED